libcommon/touch.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2023 - Tillitis AB
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+#include <tkey/led.h>
+#include <tkey/touch.h>
+
+// clang-format off
+static volatile uint32_t *touch = (volatile uint32_t *)TK1_MMIO_TOUCH_STATUS;
+// clang-format on
+
+// wait_touch_flash waits for a touch even blinking ledvalue for
+// loopcount, then restores the original LED value with origled.
+void wait_touch_flash(int ledvalue, int loopcount)
```

Comment:
Think we should use a better naming convention where touch comes first, since it is located in touch.c, an example is `touch_wait_flash()`.